devicon.json

```diff
@@ -673,8 +725,20 @@
         "color": "#59407f",
         "aliases": []
     },
+    {
+        "name": "bulma",
+        "altnames": [],
+        "tags": ["css", "framework"],
+        "versions": {
+          "svg": ["plain"],
+          "font": ["plain"]
+        },
+        "color": "#00d1b2",
+        "aliases": []
+    },
```

Comment:
`bulma` is already added (see a couple lines below). This would add it a second time.

Reply:
Same as here: https://github.com/devicons/devicon/pull/1167#discussion_r873515186